source/index.js

```diff
@@ -184,20 +184,48 @@ module.exports = async (input = 'patch', options) => {
 			enabled: () => options.yarn === true,
 			skip: () => {
 				if (options.preview) {
-					return `[Preview] Command not executed: yarn version --new-version ${input}.`;
```

Comment:
This will also mean a different preview for yarn in case we want to supply a custom message.

Reply:
Just did that! :)

Reply:
This is already changed btw.